### DISCUSSION TITLE
when cloning the airflow, got error: invalid path

### DISCUSSION THREAD

**Discussion by drelolua** [apache/airflow#24596] [General]
git clone https://github.com/apache/airflow.git
Cloning into 'airflow'...
remote: Enumerating objects: 241653, done.
remote: Counting objects: 100% (288/288), done.
remote: Compressing objects: 100% (211/211), done.
Receiving objects: 100% (241653/241653), 168.42 MiB | 2.02 MiB/s, done.1365

Resolving deltas: 100% (182077/182077), done.
error: invalid path 'tests/www/test_logs/dag_for_testing_log_view/scheduled__2017-09-01T00:00:00+00:00/task_for_testing_log_view/1.log'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

**Comment by Taragolis** [comment]
Seems like you use a Windows Host. Not all characters supported by NTFS file system.

If am right than as workaround you can run `git config core.protectNTFS false` or better use WSL2 and clone repo there

**Reply by potiuk** [reply]
Ype. Airflow cannot be used on Windows Filesystem.

**Comment by Guang1030** [comment]
Same issue here. Did not have this problem before.

**Reply by Guang1030** [reply]
Is it better to remove this kind of file or maybe rename its name. For a programmer, sometimes you need to read the source code in your local machine. This is inconvenient for a windows user.

**Reply by potiuk** [reply]
@Guang1030  Feel free to make a PR that replaces it. I think you are one of the people who has interest in it and a way to test it? 

This is an open-source project. It is created by > 2100 contributors - mostly people like you. You can become one of them and such a change is a nice one to become one.  There is no "magical someone" that will do it - if you want something to happen, you should do it.

**Reply by kokorin** [reply]
An attempt to fix the issue: #26105

**Reply by Taragolis** [reply]
@kokorin This closed PR might help https://github.com/apache/airflow/pull/25865 - which tests related to the files.

**Reply by kokorin** [reply]
Thank you, in the moment I try to run breeze under WSL2.
